composer.json

```diff
@@ -337,6 +337,7 @@
                 "Issue #3335841 - Admin toolbar menu z-index increase": "https://www.drupal.org/files/issues/2023-02-06/3335841-2.patch"
             },
             "drupal/ckeditor5_font": {
+                "Issue #3368736: no config schema' Error after update to Drupal 10.1.0": "https://www.drupal.org/files/issues/2024-01-11/ckeditor5_font-schema-fix-10.2.1-3368736-37.patch",
```

Comment:
Let's always respond in drupal.org issue queue the patch fixed issue for us with setting it to RTBC, or adding comment +1 RTBC if it is already set
/cc @rollki @aleevas

Reply:
https://www.drupal.org/node/3368736

Reply:
@podarok 
Isn't it better to leave a comment after testing? Is it enough that the developer checked it?

Reply:
Developer tested on local machine? @rollki 
It is enough